### ISSUE TITLE
Date includes time

### ISSUE THREAD

**Issue by fritut08** [mitchcapper/KPEntryTemplates#76]
When I create a child of a template with a "Date" field, the field will not only contain the date but also a time. This is not visible in the template tab but one can see it on the advanced tab.

**Comment by tgardner42** [comment]
The reverse is also true: children of a template with a Time field will contain the date as well when saved. Related, while the field is formatted for time the picker is for dates.

KeePass v. 2.51.1 (64-bit), KPET v. 8.0.0.0

**Comment by mitchcapper** [comment]
Yeah right now it uses one format and picker I could manually parse and construct date/time values rather than just tostring to potentially give you as mentioned.  Would need a set of cascading parse attempts incase you changed from one to another you wouldnt lose data.

**Comment by mitchcapper** [comment]
The picker right now is also always the same,  if someone makes a new picker control between here and #79 that has the following:
Different selectors for time/date/datetime an optional checkbox that is hidden based on another field on the control I set , and takes a nullable datetime object I can take care of the rest of the issues here and in #79.
